jupyterhub/metrics.py

```diff
@@ -30,49 +32,66 @@
 from . import orm
 from .utils import utcnow
 
+metrics_prefix = os.getenv('JUPYTERHUB_METRICS_PREFIX', 'jupyterhub')
```

Comment:
It makes more sense to me to keep `jupyterhub_` in the metric names and make the default metrics prefix `''` (same as the default 'namespace' value). What do you think? i.e. custom prefix is _in addition to_ `jupyterhub_`, not _instead of_.

Reply:
In the context of projects like [ml-tooling/ml-hub](https://github.com/ml-tooling/ml-hub) having the freedom to define the metrics prefix could be helpful for maintaining consistency within their ecosystem. While keeping `jupyterhub_` as a constant identifier in the metric names can provide a clear origin, offering full control over the metrics prefix could indeed enhance the value of this feature. Users can redefine it completely or set options like `jupyterhub_prod` and `jupyterhub_dev`. It is also common for the application name to be the namespace as the Prometheus documentation suggests.

Unfortunately, this particular option stands out as one of the limited settings that cannot be configured by Traitlets.